content/docs/command-reference/remote/modify.md

```diff
@@ -440,7 +440,7 @@ more information.
 - `url` - remote location URL.
 
   ```dvc
-  $ dvc remote modify myremote url ssh://user@example.com:1234/path/to/remote
+  $ dvc remote modify myremote url ssh://user@example.com:1234/absolute/path
```

Comment:
BTW @efiop HDFS doesn't have this config param? `url` I mean (it's not in the doc currently). Thanks

Reply:
All remotes do. That's what `dvc remote add` sets.

Reply:
Will address this in #1680.